packages/combinators/eca.js

```diff
@@ -1,18 +1,5 @@
-/*
- * Copyright 2015-2016 IBM Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Licensed to the Apache Software Foundation (ASF) under one or more contributor
+// license agreements; and to You under the Apache License, Version 2.0.
```

Comment:
The are 3 things. 
1. The IBM copyright is being removed and now this files are Apache foundation copyright. 
2. This file is apache 2.0 licensed and continues to be apache 2 licensed 
3. The header with the license it's being updated to use the standard header used for ASF projects on source and across all the OpenWhisk repos